### PR TITLE
style: cleans up code

### PR DESCRIPTION
**Pull request by jesselpalmer**
Cleans up e2es style a little to conform closer to the [Angular Style Guide](https://angular.io/styleguide).

### CODE REVIEW

File: src/e2e-app/button/button-e2e.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import {Component} from '@angular/core';
+import { Component } from '@angular/core';
```
Comment (pkozlowski-opensource) [inline src/e2e-app/button/button-e2e.ts]: Why on earth this is in the style guide??? This is just a formatting choice that each project should make on its own. As long as formatting is consistent it should be none of style guide business....

Comment (jelbourn) [inline src/e2e-app/button/button-e2e.ts]: If this is in the Angular style guide, we're not going to follow it.

File: src/e2e-app/tabs/tabs-e2e.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+import { Component } from '@angular/core';
+
 
 @Component({
   moduleId: module.id,
   selector: 'tabs-e2e',
   templateUrl: 'tabs-e2e.html',
 })
-export class BasicTabs {}
+export class TabsE2E {}
```
Comment (jelbourn) [inline src/e2e-app/tabs/tabs-e2e.ts]: I'm actually planning on renaming all of these the other direction. I dislike calling them `XxxE2e` and want the same to reflect what the test is, e.g. `BasicTabs`, `TabsInDialog`, etc.

**Comment by jelbourn** [comment]
Going to keep these things as-is.

**Comment by jesselpalmer** [comment]
@jelbourn Works for me.

**Pull request closed without merging**